src/VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -518,6 +517,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             // Use app insights telemetry 
             app.UseAppInsightsTelemetry();
+
+            
+            var mvcJsonOptions = app.ApplicationServices.GetService<IOptions<MvcNewtonsoftJsonOptions>>();
+            mvcJsonOptions.Value.SerializerSettings.Converters.Add(new PolymorphJsonConverter());
+            PolymorphJsonConverter.RegisterTypeForDiscriminator(typeof(PermissionScope), nameof(PermissionScope.Type));
+            PolymorphJsonConverter.RegisterTypeForDiscriminator(typeof(SearchCriteriaBase), nameof(SearchCriteriaBase.ObjectType));
```

Comment:
Remove

Reply:
SearchCriteriaBase is a platform class

Reply:
because it wasn't registered before

Reply:
https://github.com/VirtoCommerce/vc-platform/pull/2191